Source/Core/Gen4/Encounters4.cpp

```diff
@@ -226,6 +244,25 @@ namespace Encounters4
             }
         }
 
+        std::vector<EncounterArea4> getBugCatchingContest(const std::vector<u8> &data, const PersonalInfo *info)
```

Comment:
This needs to be passed profile so we can include/exclude based on national dex

Reply:
Done